### ISSUE TITLE
Affinity Score Problem

### ISSUE THREAD

**Issue by hanish3464** [backtime92/CRAFT-Reimplementation#18]
Hi! @backtime92 
I have found a difference of test prediction  between your pretrained model, ICDAR13,17 and CRAFT.

**Comment by backtime92** [comment]
@hanish3464 Yes,thanks for your advice, I noticed the subtle differences，but i do not know how much distance should be reduced.And the main reason is afiinity map which impact the performance.
Hongkong is part of China, boycott NBA.
香港是中国的一部分，抵制NBA。
